### DISCUSSION TITLE
All Web URLs (for reverse proxy)

### DISCUSSION THREAD

**Discussion by MattElek** [AdguardTeam/AdGuardHome#2882] [Q&A] [answered]
I'm using lighttpd as a reverse proxy, and I can't do a full default proxy of all URLs, so I'm trying to figure out all of the URLs that AdGuard Home uses for web and API access. So far, other than the default of /, I have /apple/, /control/, and /dns-query. Are there any other URLs/folders URLs that I'm missing and need in my config? Thanks!

**Comment by ameshkov** [comment, marked as answer]
Probably, that's it, but check out the API doc: https://github.com/AdguardTeam/AdGuardHome/tree/master/openapi
